### DISCUSSION TITLE
Support for Atlassian OAuth-Provider

### DISCUSSION THREAD

**Discussion by backface** [langfuse/langfuse#3986] [Ideas]
### Describe the feature or potential improvement

Could you add support for Atlassian Oauth Provider?

Should be easy as next-auth.js support it: https://next-auth.js.org/configuration/providers/oauth#built-in-providers

**Comment by marcklingen** [comment]
@backface thanks for sharing this idea! Open for a PR on this as this would be a great addition to Langfuse. Currently I'm a bit backlogged, otherwise I'd quickly add this myself
